<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-9BNFEB8" failures="1" tests="1" name="Default test(failed)" time="9.785" errors="0" timestamp="2023-04-26T23:23:18 NZST">
  <testcase classname="com.banking.Digi.TestNGAsserts2" name="TestMethodOne" time="1.027">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at com.banking.Digi.TestNGAsserts2.TestMethodOne(TestNGAsserts2.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TestMethodOne -->
  <testcase classname="com.banking.Digi.TestNGAsserts2" name="TestMethodThree" time="0.0">
    <ignored/>
  </testcase> <!-- TestMethodThree -->
  <testcase classname="com.banking.Digi.TestNGAsserts2" name="TestMethodFour" time="0.0">
    <ignored/>
  </testcase> <!-- TestMethodFour -->
  <testcase classname="com.banking.Digi.TestNGAsserts2" name="TestMethodTwo" time="0.0">
    <ignored/>
  </testcase> <!-- TestMethodTwo -->
</testsuite> <!-- Default test(failed) -->
